apps/test/integration/util/runLevelTest.js

```diff
@@ -139,7 +121,7 @@ function runLevel(app, skinId, level, onAttempt, testData) {
     firebaseAuthToken: 'test-firebase-auth-token',
     isSignedIn: true,
     isAdmin: true,
-    onFeedback: finished.bind(this),
+    onFeedback: () => finished(),
```

Comment:
why not just `onFeedback: finished,`?

Reply:
Because it's important to pass no arguments to the `done` callback.  Maybe I should express that here as
```js
() => finished(/* no args */)
```
or up above as
```js
const finished = _.once(() => done(/* no args */));
```
Do you prefer one of these, or have another idea?

Reply:
Ahh, gotcha. The latter, how it appears you did it, looks ideal